src/main/Domain.Model/Avatars/AvatarItem.cs

```diff
@@ -18,7 +19,7 @@ public class AvatarItem
         public d23Settings d23 { get; set; } = new();
         public AvatarNetworkSettings Network { get; set; } = new();
 
-        public AvatarItem(string id, string name)
+        public AvatarItem(string id, string name, string specifiedName = null)
```

Comment:
I get some weird compiler errors when I update the test references so I temporarily added a default null value.